lib/carrierwave/mounter.rb

```diff
@@ -110,7 +110,7 @@ def blank?
     end
 
     def remove?
-      remove.present? && remove !~ /\A0|false$\z/
+      remove.present? && remove.to_s !~ /\A0|false$\z/
```

Comment:
Thanks for the PR, could you make this same as the fix in the master branch?
https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mounter.rb#L115

Reply:
Done at 711d61290afcb85303068d4fb4481de7ba9345a2 😄 Thanks!